source/main/datatypes/node_t.h

```diff
@@ -8,7 +8,7 @@
 #pragma once
 
 #include <OgrePrerequisites.h>
-#include <OGRE/OgreVector3.h>
+#include <OgreVector3.h>
```

Comment:
No, because RelPosition, AbsPosition, Velocity, Forces and smoothpos are Ogre::Vector3